app/frontend/src/components/Clients.vue

```diff
@@ -0,0 +1,113 @@
+<template>
+  <v-card>
+    <template>
+      <v-card-title>
+        Registered Service Clients
+        <v-spacer></v-spacer>
+        <v-text-field
+          v-model="search"
+          append-icon="mdi-magnify"
+          label="Search"
+          single-line
+          hide-details
+        ></v-text-field>
+      </v-card-title>
+
+      <v-data-table
+        class="kc-table"
+        dense
+        :headers="headers"
+        :items="serviceClients"
+        :items-per-page="5"
+        :search="search"
+        :loading="loading"
+        loading-text="Loading... Please wait"
+      >
+        <template v-slot:item.dev="{item}">
+          <v-icon v-if="item.dev === 1" color="green">check</v-icon>
```

Comment:
Isn't the ennironment a property of the client object itself?
https://getok-dev.pathfinder.gov.bc.ca/pr-149/api/v1/docs#operation/kcServiceClientsGet
So should be able to just check the environment === 'DEV' rather than the setup of adding a flag to each one.

Reply:
Actually disregard this above.
However, should use true/false if going this way rather than a 1

Reply:
See my comment below about using a reduce/map, then you can check for each of these if the client.environments contains DEV, etc